### ISSUE TITLE
Hurricane icons do not have dates (v0.2.19)

### ISSUE THREAD

**Issue by PhillipsOwen** [RENCI/TerriaMap#60]
there are no dates shown next to the hurricane icons on the UI.

**Comment by lstillwe** [comment]
This is not and available option in styling the GeoJson layers in TerriaMap. The user can click on a hurricane icon to get the date info. In the future, I would like to write some custom code that displays the associated info when you hover the mouse over the icon - if possible.

**Comment by PhillipsOwen** [comment]
Interesting. they show up when selected/shown on the prod UI V1.

**Comment by lstillwe** [comment]
Yes - that is because there are actual layers on GeoServer for the hurricane track layers, much easier to style those there than what is available for GeoJson that I am retrieving from MetGet.
